Hosts/VS/TddStudioPackage/PackageCommands.cs

```diff
@@ -55,13 +55,23 @@ private void ExecuteChangeTddStud10State(object sender, EventArgs e)
             if (EngineLoader.IsEngineEnabled())
             {
                 EngineLoader.DisableEngine();
+                SetTddStudioDisable(true);
             }
             else
             {
                 EngineLoader.EnableEngine();
+                SetTddStudioDisable(false);
             }
         }
 
+        private void SetTddStudioDisable(bool isDisabled)

```

Comment:
What happens to the taskbar icon? What state does it go to when we disable tddstud10? Going back to the logical event model this would have been the 3rd consumer of the disabled event.